### PR TITLE
Navbar router working for ClusterView/NetworkPolicyView. NetworkPolicyView done.

### LINKED ISSUE #92: render network policies that are uploaded by user

### PR DESCRIPTION
**Pull request by joelparkk**

### CODE REVIEW

**Review by brkg — CHANGES_REQUESTED** [review]
Why is App.tsx props of type any?

File: src/components/views/NetworkPolicyView.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    })
+    if(networkPolicies.length === 1) setNetworkPolicy(networkPolicies[0].label);
+    setNetworkPoliciesArr(networkPolicies);
+  };
+  function parseData(json: any)
+  {
+    const data = dataParser.parseData(json);
+    return data;
+  }
+  const populateArray = (array: any[]): void => {
```
Comment (brkg) [inline src/components/views/NetworkPolicyView.tsx]: Is there a better type for the array parameter here?

**Review by brkg — COMMENTED** [review]
Nice use of classes here with ingress/egress/networkPolicy

File: src/components/views/NetworkPolicyView.tsx (hunk 1 above)
Comment (joelparkk) [reply src/components/views/NetworkPolicyView.tsx]: fixed !

**Comment by joelparkk** [comment]
Addressed #98 and #81

**Review by brkg — APPROVED** [review]
LGTM

**Pull request merged**